### PR TITLE
Release v3.0.0

### PR DESCRIPTION
**Pull request by PawelLipski**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -196,8 +227,7 @@ jobs:
                 command: |
                   source scripts/utils.sh
                   tag=v$(extract_version_from_current_wd)
-                  # Newlines (<br/>) are necessary for better layout on Marketplace, but make the formatting worse on GitHub
-                  change_notes=$(sed '/^<br.>$/d' CHANGE-NOTES.html)
+                  change_notes=$(./gradlew getChangelog --no-header -q)
```
Comment (PawelLipski) [inline .circleci/config.yml:230]: Actually @MaciejG604 can it be simplified even more? `--no-header` skipped here, and just `$change_notes` with no `$tag` prefix passed as `$message`?

File: CHANGE-NOTES.md [hunk 2]
```diff
@@ -0,0 +1,165 @@
+# Changelog
+
+## Unreleased
```
Comment (PawelLipski) [inline CHANGE-NOTES.md:3]: To be bumped on my pre-release PR

File: .circleci/config.yml (hunk 1 above)
Comment (MaciejG604) [reply .circleci/config.yml:230]: Unfortunately no, this is what gets created: [Draft release](https://github.com/VirtusLab/git-machete-intellij-plugin/releases)

Comment (PawelLipski) [reply .circleci/config.yml:230]: ☹️

**Pull request merged**